.gitmodules

```diff
@@ -20,7 +20,7 @@
 	fetchRecurseSubmodules = no
 [submodule "external/usersim"]
 	path = external/usersim
-	url = https://github.com/microsoft/usersim.git
+	url = https://github.com/lmb/usersim.git
```

Comment:
Should switch this back after the usersim PR is merged and before this PR is merged